scripts/trivia.js

```diff
@@ -849,9 +850,59 @@ TriviaGame.prototype.finalizeAnswers = function () {
             }*/ //This code was just causing winners to be out of order.
         }
     }
+    if (answeredCorrectly.length === totalPlayers && this.scoreType !== "elimination") {
+        var allRight = true;
+        for (var z2 = 0; z2 < leaderboard.length; z2++) {
+            leaderboard[z2][0] = answeredCorrectly[z2].name;
+            leaderboard[z2][1] = this.player(answeredCorrectly[z2].name).points;
+        }
+    }
     leaderboard.sort(function (a, b) {
         return b[1] - a[1];
     });
+    var n = 1;
```

Comment:
This definitely isn't pretty, but it works. I couldn't find a good way to sort leaderboard by time directly, so I used answeredCorrectly to sort it since answeredCorrectly is already sorted by time.